woocommerce/woocommerce
# Change WooPayments and WooPay logos in payment providers list

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #53437 .

As part of the [Woo 2.0 rebrand](https://github.com/Automattic/woopay/issues/2971), the payment providers section needs to be updated with the new logos. 
The updates required include:

1. the suggestion icon (SVG) for WooPayments
2. the WooPay icon used in the PMs Logos component.

Design: [9Mk09c2GvJBVRW6hwjlOFo-fi-247_15719](https://href.li/?https://www.figma.com/file/9Mk09c2GvJBVRW6hwjlOFo/?node-id=247-15719) 

**This change should land in WC 9.7, that will be released on February 24, 2025.**



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. install woocommerce locally and synced within WooPayments. See Pe4Hk0-3f-p2 for details.
2. enable the feature flag to see reactified settings page. pe4Hk0-1ly-p2.
3. without an account onboarded, go to `Woocommerce -> Settings -> Payments` and check the WooPayments logo and the WooPay icon in the PMs logos component

<img width="1676" alt="Screenshot 2024-12-26 at 00 57 49" src="https://github.com/user-attachments/assets/b0871065-e1e7-417e-9683-62e3d78ff18b" />


3. checkout this branch and run again `pnpm i && pnpm run build`
4. refresh the page and check the new WooPayments logo and the WooPay icon in the PMs logos component.
<img width="1685" alt="Screenshot 2024-12-26 at 00 52 38" src="https://github.com/user-attachments/assets/cc94c684-4aec-4a94-86f4-2f22bdff2689" />



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Replace Woopayments and WooPay logos with the new rebranded logos.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
